### ISSUE TITLE
Android TV app issue with resume on grouped versions

### ISSUE THREAD

**Issue by mattmarsden1978** [jellyfin/jellyfin-androidtv#1849]
### Describe the bug

If I have 2 versions of the same movie grouped together e.g. 4K and 3D versions - one of the vesions won't save the resume point. I can see it when I stop the movie and it goes to the movie info screen but when I go off that and back on it theres no resume point. If I play the other version of the movie it saves the resume point. Can this be fixed? Thanks

### Application version

0.13

### Where did you install the app from?

Sideloaded APK

### Device information

Zidoo

### Android version

Android 9

### Jellyfin server version

10.8.1

**Comment by mattmarsden1978** [comment]
Sorry to be a pain - do you know if this will be looked at as the web version works fine with resuming at the same point from grouped files, just the Android TV version that doesn't. Thanks very much.

**Comment by Andy2244** [comment]
Just to confirm the wanted logic or what the webclient does.
So all versions, including the main share the same resume point?
So if you watch the 3d version and switch to any other, it should resume from the 3d versions resume point?

**Comment by mattmarsden1978** [comment]
Yes, the same logic as the webclient - all versions resume from the same point, and yes to the PS

**Comment by Andy2244** [comment]
@nielsvanvelzen fixed in my fork and will setup a PR next week

**Comment by Janhouse** [comment]
> @nielsvanvelzen fixed in my fork and will setup a PR next week

Did you make the PR?

**Comment by Andy2244** [comment]
> Did you make the PR?

no, but should be a change i can upstream after i have a clean "lite" version working and than slowly splitting out worthwhile changes. 
Realistically will try up-streaming some of my changes early next year.  Just started working on stuff again.
